client/src/components/Users/Auth/SignUpCitizen/StepFour.tsx

```diff
@@ -48,24 +66,24 @@ export default function StepFour({ initData, handleBack, handleNext }: TStepFour
         </Typography>
         <Grid item xs={12} sx={{ height: '50px' }} />
         <Grid container item xs={12} lg={6} alignItems="center">
-          <Grid item xs={3}>
+          <Grid item xs={12}>
             <Avatar sx={{ bgcolor: 'gray', width: 110, height: 110 }} />
           </Grid>
-          <Grid item xs={3}>
-            <input accept="image/*" hidden id="upload-file" type="file" />
-            <label htmlFor="upload-file">
-              <Button
-                sx={{
-                  backgroundColor: '#EF6A60',
-                  color: 'white',
-                  borderRadius: '4px',
-                  padding: '10px',
-                }}
-                color="primary"
-              >
-                Upload
-              </Button>
-            </label>
+          <Grid item xs={12}>
+            <FileUploadInput
+              label=""
+              id="upload-file"
+              text="Upload"
+              onChange={handleImageChange}
+              buttonVariant="primaryCTAButton"
+            ></FileUploadInput>
+          </Grid>
+          <Grid item xs={12}>
+            {imageError === 'too-big' ? (
+              <Alert severity="error">Please upload a file up to 1 megabyte in size.</Alert>
+            ) : imageError === 'unsupported-ext' ? (
+              <Alert severity="error">Supported files: jpeg, jpgm png and gif</Alert>
```

Comment:
Is `jpgm` maybe a typo? (maybe meant to be `jpg`)

Reply:
it's a typo! Fixed!